code/ui/blocks/src/blocks/Canvas.stories.tsx

```diff
@@ -84,34 +80,208 @@ export const WithAdditionalActions: Story = {
   },
 };
 
-const ClassNameStoryDescripition = () => (
+export const PropSourceStateShown: Story = {
+  args: {
+    of: ButtonStories.Primary,
+    sourceState: 'shown',
+  },
+};
+
+export const PropSourceStateHidden: Story = {
+  args: {
+    of: ButtonStories.Primary,
+    sourceState: 'hidden',
+  },
+};
+
+export const PropSourceStateNone: Story = {
+  args: {
+    of: ButtonStories.Primary,
+    sourceState: 'none',
+  },
+};
+
+export const PropLayoutFullscreen: Story = {
+  args: {
+    of: ButtonStories.Primary,
+    layout: 'fullscreen',
+  },
+};
+
+export const PropLayoutCentered: Story = {
+  args: {
+    of: ButtonStories.Primary,
+    layout: 'centered',
+  },
+};
+
+export const PropLayoutPadded: Story = {
+  args: {
+    of: ButtonStories.Primary,
+    layout: 'padded',
+  },
+};
+
+export const PropSource: Story = {
+  args: {
+    of: ButtonStories.Primary,
+    source: {
+      language: 'html',
+      code: '<button>           Button          </button>', // spaces should be removed by the prettier formatter
+      format: 'html',
+    },
+  },
+};
+
+export const PropInlineStory: Story = {
+  args: {
+    of: ButtonStories.Primary,
+    story: { inline: false, height: '200px' },
+  },
+};
+
+export const PropAutoplayingStory: Story = {
+  args: {
+    of: ButtonStories.Clicking,
+    story: { autoplay: true },
+  },
+};
+
+const ClassNameStoryDescription = () => (
   <p>
     This story sets the <code>className</code> prop on the <code>Canvas</code> to{' '}
     <code>my-custom-classname</code>, which will propagate to the preview element. To demonstrate
     this, it also adds a <code>style</code> tag that sets another background color for that class:
   </p>
 );
-/**
- * This is a comment on classname
- */
-export const ClassName: Story = {
-  name: 'ClassName',
+export const PropClassName: Story = {
+  name: 'Prop ClassName',
   args: {
+    of: ButtonStories.Primary,
     className: 'my-custom-classname',
   },
   render: (args) => (
     <>
-      <ClassNameStoryDescripition />
+      <ClassNameStoryDescription />
+      <style>
+        {`
+          .my-custom-classname {
+            background-color: #fd5c9355;
+          }
+        `}
+      </style>
+      <Canvas {...args} />
+    </>
+  ),
+};
+
+export const ParameterWithToolbar: Story = {
+  name: 'parameters.docs.canvas.withToolbar = true',
```

Comment:
what's your thoughts on this naming scheme @tmeasday, as a way to distinguish between props and parameters?

![image](https://user-images.githubusercontent.com/5678122/213943322-b6cdfd31-8d83-4842-9a37-e415ac0ddda9.png)